Bump flow to v0.85

Blasz opened the pull request:
Closes #1270

ZhangYiJiang commented on www/flow-typed/package-dep-libdefs.js:
```diff
@@ -8,6 +8,14 @@ declare module '@material/snackbar' {
   declare module.exports: any;
 }
 
+declare module 'bootstrap' {
```
Please revert the changes to this file. It's automatically generated by a script which I don't have time to fix, but in short this change is unwanted. Sorry!

ZhangYiJiang commented on www/src/js/views/components/KeyboardShortcuts.jsx:
```diff
@@ -139,13 +139,19 @@ export class KeyboardShortcutsComponent extends PureComponent<Props, State> {
+
+  renderShortcut = (
+    shortcut: Shortcut,
+  ): Element<ElementType> | Array<Element<ElementType> | string> => {
```
I'm not entirely sure splitting the functions is necessary, but render functions should be typed `Node` (imported from React), which encompasses everything React accepts from `render()`.

Blasz replied:
Ah true, completely forgot about `Node`!

ZhangYiJiang commented on www/src/js/views/components/cors-stats/CorsRound.jsx:
```diff
@@ -72,7 +72,7 @@ function renderBidTable(groupedStats: GroupedBiddingStat[]) {
  * by studentType.
  */
-function CorsRound(props: Props) {
+function CorsRound(props: Props): Element<ElementType> | Element<ElementType>[] {
```
As above, type this as `Node` instead

ZhangYiJiang commented on www/src/js/views/components/cors-stats/CorsSummary.jsx:
```diff
@@ -21,7 +21,7 @@ const STUDENT_TYPE_LABELS = {
  * to get a module for each type of student in each faculty
  */
-export default function(props: Props) {
+export default function(props: Props): Element<ElementType>[] {
```
And this too

Blasz commented:
I've addressed the feedback. Fixing the prettier violation broke flow types, it couldn't seem to handle the spacing (which I think is incorrect anyway, you may need to install/update a flow prettier plugin to disable this rule) so I changed it to use a return type.
